src/krylov/gmres.c

```diff
@@ -31,7 +31,8 @@ hypre_GMRESFunctionsCreate(
    HYPRE_Int    (*Matvec)        ( void *matvec_data, HYPRE_Complex alpha, void *A,
                                    void *x, HYPRE_Complex beta, void *y ),
    HYPRE_Int    (*MatvecDestroy) ( void *matvec_data ),
-   HYPRE_Real   (*InnerProd)     ( void *x, void *y ),
+   HYPRE_Int    (*InnerProd)     ( void *x, void *y, HYPRE_Int *num_tags_ptr,
```

Comment:
It would be nice if this could be toggled on/off based on print_level, but I can't think of how to do this nicely (yet).

Reply:
I agree, but I can't think of another way either :/